blog.qmd

```diff
@@ -0,0 +1,228 @@
+---
+title: "A Deep Dive into Course Descriptions. Using Quanteda to Identify Work-Based Learning Opportunities"
+subtitle: "Data@Urban Draft"
+author-title: "Authors"
+authors: "Manuel Alcalá Kovalski and Judah Axelrod"
+affiliation: "Urban Institute"
+date: today
+format:
+  html:
+    theme: urbn.scss
+    self-contained: true
+    code-fold: true
+    code-line-numbers: true
+    html-math-method: katex
+    df-print: default
+    toc: true
+    toc-depth: 3
+    number-sections: true
+    number-depth: 3
+    highlight-style: pygments
+editor_options: 
+  chunk_output_type: console
+execute:
+  warning: false
+  message: false
+
+---
+
+```{r}
+#| include: false
+librarian::shelf(glue, gt)
+my_theme <- function(data, ...){
+  data %>%
+    opt_row_striping() %>%
+    opt_all_caps() %>%
+    opt_table_font(
+      font = list(
+        gt::google_font("Lato"),
+        default_fonts()))  %>%
+    tab_style(
+      locations = cells_title(groups = "title"),
+      style     = list(
+        cell_text(weight = "bold", size = 24)
+      )
+    ) %>%
+    #Apply different style to the title
+    tab_style(
+      locations = cells_title(groups = "title"),
+      style     = list(
+        cell_text(weight = "bold", size = 24)
+      )
+    ) %>%
+    tab_options(
+      column_labels.border.top.width = px(5),
+      column_labels.border.top.color = "#FFFFFF",
+      table.border.top.color = "#FFFFFF",
+      table.border.bottom.color = "#FFFFFF",
+      heading.background.color = '#1696d2',
+      data_row.padding = px(5),
+      source_notes.font.size = 16 ,
+      heading.align = "center",
+      row_group.background.color = '#D0D3D4',
+      ...)
+}
+
+```
+
+
+*This blog post is part two of a series on analyzing work-based learning
+opportunities in community colleges. In [part
+one](https://urban-institute.medium.com/how-web-scraping-powered-our-analysis-of-work-based-learning-opportunities-in-community-colleges-3452320dc345),
+we discussed how we used web scraping to gather course descriptions from
+community colleges in Florida. Now, we'll delve into how we analyzed this data
+using the quanteda package in R.*
+
+## Introduction
+
+In our previous post, we detailed our journey of collecting course descriptions
+from Florida's community colleges using web scraping techniques. We successfully
+compiled a comprehensive dataset, but the question remained: how do we make
+sense of this vast amount of text data? Enter quanteda, an R package designed
+for quantitative text analysis.
+
```

Comment:
Love the setup - I think a little more info about what we hoped to do with quanteda would be helpful here. Maybe instead of the question you pose here, it could be more specific about what the research goal was of the text analysis? Like "how can we understand more about the prevalence of WBL opportunities in this vast amount of text data" etc.

Reply:
@alenastern